utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -32,11 +32,11 @@ class UrlImageParser private constructor(
    * @return Drawable : Drawable representation of the image.
    */
   override fun getDrawable(urlString: String): Drawable {
-    imageDownloadUrlTemplate = String.format(imageDownloadUrlTemplate, entityType, entityId, urlString)
+    val imageUrl =  String.format(imageDownloadUrlTemplate, entityType, entityId, urlString)
```

Comment:
@BenHenning Resolved duplicate pictures. Fix was to create different variable to form an URL. `imageDownloadURLTemplate` was not changing, may be because it is annotated with Singleton but not sure about it.